conf/defaults.ini

```diff
@@ -1626,3 +1626,11 @@ server_name =
 proxy_address =
 # Determines if the secure socks proxy should be shown on the datasources page, defaults to true if the feature is enabled
 show_ui = true
+
+################################## Feature Management ##############################################
+[feature_management]
+# Hides specific feature toggles from the feature management page
+hidden_toggles =
```

Comment:
Do we need these as instance level config? (Vs just options in each feature)

Reply:
These configs were proposed during our DNA discussion, as they could be useful for operators (HG or others) to decide which features their grafana administrators would be able to tune in.